src/test/java/org/apache/commons/collections4/list/FixedSizeListTest.java

```diff
@@ -83,4 +83,66 @@ public void testListAllowsMutationOfUnderlyingCollection() {
         Assert.assertEquals("Modifying an the underlying list is allowed",
                 sizeBefore + 1, fixedSizeList.size());
     }
+
+    //-----------------------------------------------------------------------
```

Comment:
Let's not add these kinds of `//---...` lines as they appear, to me, as just noise.

Reply:
Ok, I have removed all the new ones, thank you for your suggestion.